msi/release.sh

```diff
@@ -176,20 +176,24 @@ BUILD_EXTRA_WINPATH="$(cd "$SCRIPT_PATH"/.. && pwd -W | tr / \\\\)"
             <Component Directory="INSTALLFOLDER:\\usr\\share\\git\\">
                 <File Id="ReleaseNotes_Css" Source="$BUILD_EXTRA_WINPATH\\usr\\share\\git\\ReleaseNotes.css" />
             </Component>
+            <Component Directory="ApplicationFolder">
+                <File Id="GfwGitConfig" Name="config" Source="mingw$BITNESS\\etc\\gitconfig" />
+            </Component>
 EOF
 echo "$LIST" |
 sort |
 uniq |
 tr / \\\\ |
-sed -e 's/\(.*\)\\\(.*\)/            <Component Directory="INSTALLFOLDER:\\\1\\">\
+sed -r -e "/^mingw$BITNESS\\\\etc\\\\gitconfig/ d"\
+        -e 's/(.*)\\(.*)/            <Component Directory="INSTALLFOLDER:\\\1\\">\
                 <File Source="&" \/>\
             <\/Component>/' \
-        -e 's/^\([^\\]*\)$/            <Component Directory="INSTALLFOLDER">\
+	    -e 's/^([^\\]*)$/            <Component Directory="INSTALLFOLDER">\
```

Comment:
Please move this into a separate commit, or fix up the commit that introduced the bug (if it is part of this Pull Request).